packages/actor-query-source-identify-hypermedia/test/QuerySourceHypermedia-test.ts

```diff
@@ -741,101 +741,31 @@ describe('QuerySourceHypermedia', () => {
         ]);
         expect(mediatorsThis.mediatorQuerySourceIdentifyHypermedia.mediate).toHaveBeenCalledTimes(3);
 
-        expect(it1Meta).toHaveBeenCalledTimes(4);
```

Comment:
Can you motivate these changes?
Is it because the number of calls can differ across runs?
If it not, I would suggest keeping the separate checks per call (and nr of times).

Reply:
During my testing, I **think** that noticed that for iterator 2, the number of calls varied. Also, it seemed like what really mattered was the final cardinality estimate, rather than how many times the metadata changed. I will  spend more time digging into whether those specific counts actually make sense. If I remember correctly, `it2` was called between 8 and 10 times, which I was not able to understand why.

Reply:
For the intermediary cardinality, I was also not really able to understand why at some very specif point it was evaluated (look iterator 2) ad how it was determined that it should be those specific values.

Reply:
I will follow up later for my first point. But my second point stand without further investigation (I mean it can be wrong, but I do not think I need to investigate more to be sure of my comment)

Reply:
As discussed in person the number of metadata update can vary with this implementation.